packages/common/validation.js

```diff
@@ -48,6 +47,9 @@ export const validateForm = (fValues, fSchema) => {
           validateForm1(values[v], schema[v], collector);
         }
       });
+    if (values.password && values.passwordConfirmation && values.password !== values.passwordConfirmation) {
```

Comment:
Why is this checker here? What happened when comparable fields have different names?